src/tintx/reader.py

```diff
@@ -248,7 +254,12 @@ def grids(self) -> Iterator[GridType]:
     @property
     def tracks(self) -> pd.DataFrame:
         """Pandas ``DataFrame`` representation of the tracked cells."""
-        return self._tracks
+        import geopandas as gpd  # type: ignore
+        from shapely import geometry  # type: ignore
+
+        tracks = self._tracks.copy()
+        tracks["geometry"] = [geometry.shape(geom) for geom in tracks["geometry"]]
+        return gpd.GeoDataFrame(tracks, crs=self.crs)
```

Comment:
Wouldn't it be better to create the geopandas dataframe from the very beginning? That is adding it into the `get_object_prop` method?